src/smbclient/_os.py

```diff
@@ -1019,6 +1022,48 @@ def setxattr(path, attribute, value, flags=0, follow_symlinks=True, **kwargs):
         set_info(transaction, ea_info)
 
 
+class SMBFileStreamInformation(t.NamedTuple):
```

Comment:
Just a minor nitpick could you move this to the top alongside the other tuples please

Reply:
Oh, I didn't even see them there as they're using `collections.namedtuple`. Worth upgrading them to classes now? Or just leave them alone for now

Reply:
I think just leave them alone for now. Can always change to the typing variant at a future point in time.